lib/datadog/tracing/configuration/settings.rb

```diff
@@ -249,6 +249,17 @@ def self.extended(base)
                 o.type :bool
               end
 
+              # Forces the tracer to always send span events with the native span events format
+              # regardless of the agent support. This is useful in agent-less setups.
+              #
+              # @default `DD_TRACE_NATIVE_SPAN_EVENTS` environment variable, otherwise `false`
+              # @return [Boolean,nil]
+              option :native_span_events do |o|
+                o.env Tracing::Configuration::Ext::ENV_NATIVE_SPAN_EVENTS
+                o.default nil
+                o.type :bool, nilable: true
+              end
```

Comment:
I think it's clear in the PR, where all of the diff shows up together what `nil` means, but when looking only at the setting definition, it's not very explicit what happens with the `nil` (and the comment for the default is actually incorrect).

TL;DR: I think it's worth updating the comment to state what happens in both `false` and `nil` situations.

(As an alternative, there's a situation in the profiler where we have this true/false/other config and [we've explicitly used `:auto`](https://github.com/DataDog/dd-trace-rb/blob/master/lib/datadog/core/configuration/settings.rb#L401) although it slightly complicates parsing the setting)

Reply:
I added documentation, nice call out!